components/meta-packages/history/history

```diff
@@ -558,6 +558,7 @@ image/editor/gimp/locale/zh_cn@0.5.11,5.11-2015.0.2.0
 image/gnome-backgrounds@2.32.0,5.11-2017.0.0.1
 image/graphviz/graphviz-perl-512@2.28.0,5.11-2015.0.1.0
 image/graphviz/graphviz-perl-516@2.28.0,5.11-2015.0.1.5
+image/library/libjpeg@6.0.2,5.11-2020.0.1.0
 image/nvidia/cg-toolkit@2.2,5.11-2015.0.2.0
```

Comment:
You'll need to increment the COMPONENT_REVISION!

Reply:
Right found it :)